helper-functions/taskProgress.js

```diff
@@ -0,0 +1,14 @@
+import { timeWas } from '../helper-functions/time-was';
+
+function percentageofDaysRemaining(startedOn, endsOn, completedDate) {
+  let startDate = timeWas(startedOn * 1000, true, endsOn * 1000);
+  let endDate = timeWas(endsOn * 1000, true, startDate * 1000);
+  const date1 = new Date(startDate);
```

Comment:
Why not giving some understandable names here ? 
Like: `start` and `end` or `preDate` and `postDate`

Reply:
> Why not giving some understandable names here ?
> Like: `start` and `end` or `preDate` and `postDate`

Added start date and end date as start and end date of the task.